roles/nginx/templates/server.conf.j2

```diff
@@ -3,7 +3,7 @@ server {
     server_name $hostname;    # e.g. box.lan, set dynamically when NGINX starts
     #server_name {{ iiab_hostname }}.{{ iiab_domain }};
     #server_name {{ iiab_hostname }};
-    listen 80;
+    listen {{ nginx_port }};
```

Comment:
#2834
Further soft-code nginx variables to be able to run as a second instance of nginx on a foreign/unknown platform.
Exposes new variable nginx_dir for alternate location use.
Supplies custom systemd file for use with nginx_dir is a non-stock location.
Allows further evaluation of the new platform with nginx on a different port